packages/react/src/index.ts

```diff
@@ -6,7 +6,15 @@ export type { ErrorBoundaryProps, FallbackRender } from './errorboundary';
 export { ErrorBoundary, withErrorBoundary } from './errorboundary';
 export { createReduxEnhancer } from './redux';
 export { reactRouterV3Instrumentation } from './reactrouterv3';
-export { reactRouterV4Instrumentation, reactRouterV5Instrumentation, withSentryRouting } from './reactrouter';
+export {
+  // eslint-disable-next-line deprecation/deprecation
+  reactRouterV4Instrumentation,
+  // eslint-disable-next-line deprecation/deprecation
+  reactRouterV5Instrumentation,
+  withSentryRouting,
+  browserTracingReactRouterV4Integration,
+  browserTracingReactRouterV5Integration,
```

Comment:
Idk if this makes sense but `reactRouterBrowserTracingVXIntegration` would sound better to me. Any reason you chose this order? ^^